Description: Fix FTBFS error: error in setup.cfg: command 'build' has no such option 'i18n'
 We did not import DistUtilsExtra
Author: Julian Andres Klode <juliank@ubuntu.com>


--- aptdaemon-1.1.1+bzr982.orig/setup.py
+++ aptdaemon-1.1.1+bzr982/setup.py
@@ -12,6 +12,11 @@ import os
 import sys
 from setuptools import *
 
+from DistUtilsExtra.command import (
+    build_extra,
+    build_i18n,
+)
+
 import aptdaemon
 
 readme = open(os.path.join(os.path.dirname(__file__), "README")).read()
@@ -32,6 +37,8 @@ setup(name="aptdaemon",
       test_suite="nose.collector",
       license="GNU LGPL",
       keywords="apt package manager deb dbus d-bus debian ubuntu dpkg",
+      cmdclass={"build": build_extra.build_extra,
+                "build_i18n": build_i18n.build_i18n},
       classifiers=["Development Status :: 5 - Production/Stable",
                    "Intended Audience :: System Administrators",
                    "License :: OSI Approved :: GNU Library or Lesser General " \
